app/templates/teachers/convert-to-teacher-account-view.jade

```diff
@@ -28,40 +28,36 @@ block content
       .row.m-y-2
         .col-md-offset-2.col-md-4.col-sm-6
           .form-group
-            label.control-label(data-i18n="general.username")
+            span.control-label(data-i18n="general.username")
```

Comment:
How come these were changed? Labels, when clicked, focus the thing they're connected to... although that wasn't set up for this anyway since you need the `for` attribute to make that happen, I think.